integration-tests/cypress/e2e/project/edit/explorer.cy.js

```diff
@@ -67,36 +67,41 @@ describe('/projects/:projectId/edit - Explorer', () => {
     cy.getByTestId('selected').contains('robot');
   });
 
+  const option = {
+    delay: 200,
+  };
+
```

Comment:
Don't use a delay, improve instead the test. This will not work all the time because sometime, what you are waiting for will arrive in 300ms.